code/modules/biomatter_manipulation/bioreactor/biotank.dm

```diff
@@ -90,7 +90,7 @@
 
 /obj/machinery/multistructure/bioreactor_part/biotank_platform/proc/take_amount(var/amount)
 	biotank.reagents.add_reagent("biomatter", amount)
-	GLOB.biomatter_neothecnology_amt += amount
+	GLOB.biomatter_neothecnology_amt += new_amount
```

Comment:
```suggestion
	GLOB.biomatter_neothecnology_amt += amount
```
makes it complie

Reply:
Why was it in the surgery commit and not the other one?